.github/CONTRIBUTING.MD

```diff
@@ -0,0 +1,102 @@
+# Contributing
+
+Hacking Prodigy used to be as easy as `PIXI.game.prodigy.player.gold=999`, but now we need a whole team!
+
+If you'd like to join as an official collaborator, you can message us on [our official Discord server](https://discord.gg/XQDfbfq)!
```

Comment:
adding a prerequisite of having coding skills would be good here

Reply:
I just copied those files from /meta, lmao

Reply:
Alr, Imma add that

Reply:
done